core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -4169,7 +4171,10 @@ private SqlDialect nonOrdinalDialect() {
     final String query = "SELECT MAP['k1', 'v1', 'k2', 'v2']";
     final String expected = "SELECT MAP ('k1', 'v1', 'k2', 'v2')\n"
         + "FROM (VALUES (0)) t (ZERO)";
-    sql(query).withSpark().ok(expected);
+    final String expectedPresto = "SELECT MAP (ARRAY['k1', 'k2'], ARRAY['v1', 'v2'])\n"
```

Comment:
Maybe we should add a comment with a jira link in the comment

Reply:
done, added new tests